### PR TITLE
Update Duplication Request Modal to support new RAC fee model

### LINKED ISSUE #680: Update DIMES Duplication Request Form to Reflect New Fee Model
## Is your feature request related to a problem? Please describe.
With the [updated fee model](https://github.com/RockefellerArchiveCenter/rockarch.org/pull/280), the DIMES request modal needs a refresh.

## Describe the solution you'd like
Update the "fee schedule" link to point to the correct heading in rockarch.org (#duplication-services), since the #duplication-services-and-fee-schedule heading went away.

## Describe alternatives you've considered
Instead of just updating the link, we could replace the "note" at the top to say something like:
"We charge a $25 flat fee per item, with a limit of 20 items per year. For details including exceptions for audiovisual materials and special formats, see our [fee schedule](url). For help, email an archivist at archive@rockarch.org."

It's also worth considering if we need the "format" dropdown menu at all, since researchers aren't really choosing a format.

If we decide to change any language, consider the need to update translations.

## Additional Context
Here's what the UI looks like now:
image

### LINKED ISSUE #682: Some times in datepicker are not fully visible
## Describe the bug
Datepicker CSS is causing times to be cut off and not fully in view of user

## To reproduce
1. Go to Reading Room Request modal in my list
2. Click on datepicker and see that times are cut off

## Error message or screenshots
image

### PR DESCRIPTION
**Pull request by HaSistrunk**
Fixes #680
Fixes #682 

- Adds new language to quickly orient researchers to the associated fee and request limit, and provides accurate links for them to find more detailed information.
- Removes select menu to choose a format, since the user does not actually choose the format.
- Updates associated translations
- Also tweaks the datepicker styles to fix issue of times not being fully visible in UI.

Note: Once merged, we'll need to update related analytics events in Matomo Analytics.

Screenshot of updated UI (note that captcha doesn't load in local build):
image

### CODE REVIEW

**Comment by HaSistrunk** [comment]
I see the Travis fail, working to update the tests now...

**Comment by helrond** [comment]
This looks good! Can you connect with Patrick on the translation question?

**Comment by HaSistrunk** [comment]
@p-galligan this is ready for translation updates.

**Comment by p-galligan** [comment]
Ok, this is ready for review with translations!

File: src/locales/fr/messages.po [hunk 1]
```diff
@@ -171,13 +171,18 @@ msgstr "<0>Informations sur la citation copiées dans le presse-papiers.</0>"
 #. Button to copy citation text
 #: src/components/RecordsDetail/index.js:215
 msgid "<0>Cite<1/></0>"
-msgstr "<0>Citer<1/></0>"
+msgstr "<0>Citer<1/></0><<<<<<< Updated upstream======="
```
Comment (HaSistrunk) [inline src/locales/fr/messages.po]: @p-galligan I'm seeing this "updated upstream" in the files.

Comment (p-galligan) [reply src/locales/fr/messages.po]: @HaSistrunk good catch. This was related to stashed changes and updates. One of the downsides of having long-lived branches hanging out.

I've removed them.

**Comment by HaSistrunk** [comment]
@helrond are we good to merge this?

**Review by helrond — APPROVED** [review]

**Pull request merged**